sky/cli.py

```diff
@@ -3736,6 +3737,7 @@ def spot_dashboard(port: Optional[int]):
 
     with subprocess.Popen(ssh_command, shell=True,
                           start_new_session=True) as ssh_process:
+        time.sleep(5)  # Added delay for ssh_command to initialize.
```

Comment:
Does 2s or 1s work? 5s feels a bit long.

Reply:
Reduced to 3s. 
It takes around 2+s on my computer. 
Or maybe we can simply remove webbrowser.open and let user click the link in the prompt.